content/en/docs/reference/glossary/kops.md

```diff
@@ -4,25 +4,29 @@ id: kops
 date: 2018-04-12
 full_link: /docs/getting-started-guides/kops/
 short_description: >
-  A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters. *NOTE&#58; Officially supports AWS only, with GCE and VMware vSphere in alpha*.
+  A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters.
```

Comment:
nit: i think we can take out `you`

Reply:
Per the style guide, I'm happy to address the reader as “you”.
Compare to:
> kubeadm helps you bootstrap a minimum viable Kubernetes cluster that conforms to best practices

from https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/

Reply:
okay ... is just didn't feel right to me...